examples/rand-extension/lib.rs

```diff
@@ -130,5 +132,39 @@ mod rand_extension {
             let rand_extension = RandExtension::default();
             assert_eq!(rand_extension.get(), [0; 32]);
         }
+
+        #[ink::test]
+        fn chain_extension_works() {
+            // given
+            struct MockedExtension;
+            impl ink_env::test::ChainExtension for MockedExtension {
+                /// The static function id of the chain extension.
+                fn func_id(&self) -> u32 {
+                    1101
```

Comment:
I'd make this a `const` since it's used a couple of times in the example already

Reply:
It's only used in one other place, where I can't replaced it with a `const`. Here:
```
#[ink(extension = FUNC_ID, returns_result = false)]
```